e2e/scripts/st_legacy_add_rows.py

```diff
@@ -151,8 +151,3 @@
             .interactive(),
             use_container_width=True,
         )
-
-# Test that _legacy_add_rows errors out when the dataframe dimensions don't
-# match. Should show an error.
-dataframe_el = st._legacy_dataframe(df1)
-dataframe_el._legacy_add_rows(np.abs(np.random.randn(num_rows, 6)))
```

Comment:
This e2e test relied on add_rows always being handled in Python (and it's therefore been a source of flakiness, because we can't guarantee this). This is the one e2e test that's removed.